app/pb_kits/playbook/pb_nav/_item.jsx

```diff
@@ -4,24 +4,34 @@ import PropTypes from "prop-types";
 const propTypes = {
   text: PropTypes.string.isRequired,
   link: PropTypes.string,
-  active: PropTypes.bool
+  active: PropTypes.bool,
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node
+  ])
 };
 
 const defaultProps = {
-  text: "Hello Item",
   link: "#",
-  active: false
+  active: false,
+  text: ''
 };
```

Comment:
With no prop-types there are no default props so you can delete `defaultProps`. See the comment below: